changelog.txt

```diff
@@ -7,6 +7,15 @@ The **signac-dashboard** package follows `semantic versioning <https://semver.or
 Version 0.3
 ===========
 
+[next release] -- YYYY-MM-DD
```

Comment:
I'd put this in a `Version 0.4` section. This feels worthy of a minor version bump.

Reply:
Will do.